src/ark/utils/plot_utils.py

```diff
@@ -987,15 +993,17 @@ def cohort_cluster_plot(
     if isinstance(cmap, pd.DataFrame):
         unique_clusters: pd.DataFrame = cmd.mapping[[cmd.cluster_column,
                                                      cmd.cluster_id_column]].drop_duplicates()
-        merged_cmap: pd.DataFrame = cmap.merge(right=unique_clusters, on=cmd.cluster_column)
-        cmap_colors = merged_cmap["color"].values
-        colors_like: list[bool] = list(map(colors.is_color_like, cmap_colors))
+        cmap_colors: pd.DataFrame = cmap.merge(
+            right=unique_clusters,
+            on=cmd.cluster_column
+        ).sort_values(by="cluster_id")["color"].values
+        colors_like: list[bool] = [colors.is_color_like(c) for c in cmap_colors]
```

Comment:
What does `is_color_like` do here?

Reply:
It checks to see if an object is a valid color. For example, it'll work with hex, rgb, rgba, named colors like "orange" and "crimson"

Reply:
gotcha